content/applications/inventory_and_mrp/quality/quality_check_types/instructions_check.rst

```diff
@@ -0,0 +1,90 @@
+==========================
+Instructions quality check
+==========================
+
+.. |QCP| replace:: :abbr:`QCP (Quality Control Point)`
+
+In Odoo *Quality*, an *Instructions* check is one of the quality check types that can be selected
+when creating a new quality check or quality control point (QCP). Instructions checks consist of a
+text entry field that allows the creator to provide instructions for how to complete the check.
+
+For a full overview of how to configure a quality check or a |QCP|, see the documentation on
+:ref:`quality checks <quality/quality_management/quality-checks>` and :ref:`quality control
+points <quality/quality_management/quality-control-points>`.
+
+Process an Instructions quality check
+=====================================
+
+There are multiple ways that Instructions quality checks can be processed. If a quality check is
+assigned to a specific manufacturing, inventory, or work order, the check can be processed on the
+order itself. Alternatively, a check can be processed from the check's page.
+
+Process from the quality check's page
+-------------------------------------
+
+To process an Instructions quality check from the check's page, begin by navigating to
```

Comment:
we're indirectly referring to an Odoo UI label in these contexts so let's italicize otherwise the capitalization (+ other formatting) isn't needed
```suggestion
To process an *Instructions* quality check from the check's page, begin by navigating to
```